zio-http/jvm/src/test/scala/zio/http/ClientHttpsSpec.scala

```diff
@@ -73,7 +73,7 @@ object ClientHttpsSpec extends ZIOHttpSpec {
           ),
         ),
       )
-    } @@ nonFlaky(20),
+    } @@ nonFlaky(20) @@ ignore,
```

Comment:
The server the test uses seems to be not working at the moment which is causing CI to fail so I disabled the test. We should open a separate ticket to use a local server instance (perhaps even zio-http now that it supports SSL) instead of relying on a remote server